src/go/k8s/cmd/main.go

```diff
@@ -377,7 +379,7 @@ func main() {
 			Client:         mgr.GetClient(),
 			EventRecorder:  helmRepositoryEventRecorder,
 			Getters:        getters,
-			ControllerName: "redpanda-controller",
+			ControllerName: "redpanda-helmRepository-controller",
 			TTL:            15 * time.Minute,
 			Metrics:        metricsH,
 			Storage:        storage,
```

Comment:
There is missing Cache and CacheRecorder.

https://github.com/redpanda-data/redpanda-operator/pull/43/files#diff-8659c00d96d07c10eb47505088baad8cb1d6563e41cfc08e532d214816a400efR400-R401